frontend/src/lib/functions.ts

```diff
@@ -5,3 +5,32 @@ export function matchAny(search: string, options: string[]): boolean {
         -1
     );
 }
+
+export function howLongAgo(time: string): string {
```

Comment:
https://www.npmjs.com/package/javascript-time-ago

Reply:
Welps 😂

Reply:
In JS / Python there's a package for anything ;)